modules/oe_content_event/src/Entity/EventVenue.php

```diff
@@ -0,0 +1,277 @@
+ *     "translation" = "Drupal\content_translation\ContentTranslationHandler",
+ *     "form" = {
+ *       "default" = "Drupal\oe_content_event\Form\EventVenueForm",
+ *       "add" = "Drupal\oe_content_event\Form\EventVenueForm",
+ *       "edit" = "Drupal\oe_content_event\Form\EventVenueForm",
+ *       "delete" = "Drupal\Core\Entity\ContentEntityDeleteForm",
+ *     },
+ *     "route_provider" = {
+ *       "html" = "Drupal\Core\Entity\Routing\AdminHtmlRouteProvider",
+ *     },
+ *   },
+ *   base_table = "event_venue",
+ *   data_table = "event_venue_field_data",
+ *   revision_table = "event_venue_revision",
+ *   revision_data_table = "event_venue_field_revision",
+ *   translatable = TRUE,
+ *   admin_permission = "administer event venue entities",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "revision" = "vid",
+ *     "label" = "name",
+ *     "uuid" = "uuid",
+ *     "uid" = "uid",
+ *     "owner" = "uid",
+ *     "langcode" = "langcode",
+ *     "published" = "status",
+ *   },
+ *   revision_metadata_keys = {
+ *     "revision_user" = "revision_uid",
+ *     "revision_created" = "revision_timestamp",
+ *     "revision_log_message" = "revision_log"
+ *   },
+ *   links = {
+ *     "canonical" = "/event_venue/{event_venue}",
+ *     "add-form" = "/admin/content/event_venue/add",
+ *     "edit-form" = "/admin/content/event_venue/{event_venue}/edit",
+ *     "delete-form" = "/admin/content/event_venue/{event_venue}/delete",
+ *     "collection" = "/admin/content/event_venue",
+ *   },
+ *   field_ui_base_route = "event_venue.settings"
+ * )
+ */
+class EventVenue extends EditorialContentEntityBase implements EventVenueInterface {
+
+  use EntityOwnerTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getName() {
+    return $this->get('name')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setName($name) {
+    $this->set('name', $name);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCapacity() {
+    return $this->get('capacity')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setCapacity($capacity) {
+    $this->set('capacity', $capacity);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getRoom() {
+    return $this->get('room')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setRoom($room) {
+    $this->set('room', $room);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCreatedTime() {
+    return $this->get('created')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setCreatedTime($timestamp) {
+    $this->set('created', $timestamp);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
+    $fields = parent::baseFieldDefinitions($entity_type);
+    $fields += static::ownerBaseFieldDefinitions($entity_type);
+
+    $fields['uid']
```

Comment:
Isn't the `ownerBaseFieldDefinitions()` defining this?

Reply:
No, it defines owner.

Reply:
Now we don't have at all usage of `ownerBaseFieldDefinitions()`